packages/view/backend/sirius-components-view-form/src/main/resources/model/form.ecore

```diff
@@ -80,6 +80,10 @@
         eType="ecore:EDataType ../../../../../sirius-components-view/src/main/resources/model/view.ecore#//Length"
         defaultValueLiteral="250"/>
   </eClassifiers>
+  <eClassifiers xsi:type="ecore:EClass" name="SplitButtonDescription" eSuperTypes="#//WidgetDescription">
+    <eStructuralFeatures xsi:type="ecore:EReference" name="actions" upperBound="-1"
+        eType="#//ButtonDescription" containment="true"/>
+  </eClassifiers>
```

Comment:
Please move this widget below, they are in alphabetical order (also please move TreeDescription, I didn't see it when I reviewed the PR for the Tree with Checkbox.